Makefile

```diff
@@ -1,55 +1,106 @@
-EYNOLLAH_MODELS ?= $(PWD)/models_eynollah
-export EYNOLLAH_MODELS
+PYTHON ?= python3
+PIP ?= pip3
+EXTRAS ?=
 
 # DOCKER_BASE_IMAGE = artefakt.dev.sbb.berlin:5000/sbb/ocrd_core:v2.68.0
-DOCKER_BASE_IMAGE = docker.io/ocrd/core:v2.68.0
+DOCKER_BASE_IMAGE = docker.io/ocrd/core-cuda-tf2:v3.3.0
 DOCKER_TAG = ocrd/eynollah
 
+#SEG_MODEL := https://qurator-data.de/eynollah/2021-04-25/models_eynollah.tar.gz
+#SEG_MODEL := https://qurator-data.de/eynollah/2022-04-05/models_eynollah_renamed.tar.gz
+SEG_MODEL := https://qurator-data.de/eynollah/2022-04-05/models_eynollah.tar.gz
+#SEG_MODEL := https://github.com/qurator-spk/eynollah/releases/download/v0.3.0/models_eynollah.tar.gz
+#SEG_MODEL := https://github.com/qurator-spk/eynollah/releases/download/v0.3.1/models_eynollah.tar.gz
+
+BIN_MODEL := https://github.com/qurator-spk/sbb_binarization/releases/download/v0.0.11/saved_model_2021_03_09.zip
```

Comment:
The binarization model is already included in `models_eynollah` from both of these downloads:
* https://github.com/qurator-spk/eynollah/releases/download/v0.3.1/models_eynollah.tar.gz
* https://qurator-data.de/eynollah/2022-04-05/models_eynollah.tar.gz

Reply:
I did compare `saved_model_2021_03_09` with `eynollah-binarization_20210425` in the current Eynollah models, but it was not the same.

So you are saying the latter _should_ be used as replacement for sbb_binarize? What should we call it (from the user side)?

Reply:
> the latter should be used as replacement model for sbb_binarize

IIUC yes (at least this was the idea to consolidate code and models), but let me quickly confirm this with @vahidrezanezhad.

Reply:
AFAICT, the model `eynollah-binarization_20210425` which is already included in `models_eynollah.tar.gz` is the only binarization model needed for both processors and thus also replaces `saved_model_2021_03_09`. 

Btw there is also an even newer hybrid binarization model [here](https://qurator-data.de/sbb_binarization/2022-08-16/model_bin_hybrid_savedmodel.tar.gz), but that first needs to be published properly, so we can ignore it for the immediate context here.